src/config.yaml

```diff
@@ -1,87 +1,87 @@
-options:
-  volume-backend-name:
-    type: string
-    default: "cinder_netapp"
-    description: Service name to present to Cinder
-  netapp-storage-family:
-    type: string
-    default: "ontap_cluster"
-    description: |
+"options":
```

Comment:
Why is everything quoted in here?

Reply:
when added layer it produces the config in quotes and this quote is to match the final config produced in charm. https://api.jujucharms.com/charmstore/v5/~narindergupta/cinder-netapp-4/archive/config.yaml

As sl_ca and other layer config were in quotes so i matches that.